[2024-02-10T04:12:58.297+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: weather_api_dag.load_data_to_postgres backfill__2024-02-09T01:00:00+00:00 [queued]>
[2024-02-10T04:12:58.396+0000] {taskinstance.py:1957} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: weather_api_dag.load_data_to_postgres backfill__2024-02-09T01:00:00+00:00 [queued]>
[2024-02-10T04:12:58.398+0000] {taskinstance.py:2171} INFO - Starting attempt 1 of 1
[2024-02-10T04:12:58.489+0000] {taskinstance.py:2192} INFO - Executing <Task(PythonOperator): load_data_to_postgres> on 2024-02-09 01:00:00+00:00
[2024-02-10T04:12:58.531+0000] {standard_task_runner.py:60} INFO - Started process 5088 to run task
[2024-02-10T04:12:58.628+0000] {standard_task_runner.py:87} INFO - Running: ['***', 'tasks', 'run', 'weather_api_dag', 'load_data_to_postgres', 'backfill__2024-02-09T01:00:00+00:00', '--job-id', '43', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/weather_api_dag.py', '--cfg-path', '/tmp/tmpxzmx1e59']
[2024-02-10T04:12:58.662+0000] {standard_task_runner.py:88} INFO - Job 43: Subtask load_data_to_postgres
[2024-02-10T04:12:59.248+0000] {task_command.py:423} INFO - Running <TaskInstance: weather_api_dag.load_data_to_postgres backfill__2024-02-09T01:00:00+00:00 [running]> on host 442b6bee1a9c
[2024-02-10T04:13:00.109+0000] {taskinstance.py:2481} INFO - Exporting env vars: AIRFLOW_CTX_DAG_EMAIL='tharathip.kul@gmail.com' AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='weather_api_dag' AIRFLOW_CTX_TASK_ID='load_data_to_postgres' AIRFLOW_CTX_EXECUTION_DATE='2024-02-09T01:00:00+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='backfill__2024-02-09T01:00:00+00:00'
[2024-02-10T04:13:00.233+0000] {logging_mixin.py:188} INFO - None
[2024-02-10T04:13:00.233+0000] {python.py:201} INFO - Done. Returned value was: None
[2024-02-10T04:13:00.257+0000] {taskinstance.py:1138} INFO - Marking task as SUCCESS. dag_id=weather_api_dag, task_id=load_data_to_postgres, execution_date=20240209T010000, start_date=20240210T041258, end_date=20240210T041300
[2024-02-10T04:13:00.472+0000] {local_task_job_runner.py:234} INFO - Task exited with return code 0
[2024-02-10T04:13:00.598+0000] {taskinstance.py:3281} INFO - 0 downstream tasks scheduled from follow-on schedule check
